src/redux/books/books.js

```diff
@@ -0,0 +1,16 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
```

Comment:
- [ ] Consider using [the duck pattern](https://github.com/erikras/ducks-modular-redux) for your action types definition.

  ```
  const ADD_BOOK = 'bookstore/books/ADD_BOOK';
  const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
  ```